### PR TITLE
Let's see what happens if we update universe

### PR DESCRIPTION
**Pull request by jsancio**
Upgrade all of the tests to use the latest Universe packages.

We needed to update the CI environments to support UTF-8 encoding because the output printed by the CLI could be encoded using UTF-8.

### CODE REVIEW

**Comment by jsancio** [comment]
@tamarrow @cruhland @larioj This is ready for review!

File: cli/tests/data/helloworld-v3-stub.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      "network": "BRIDGE"
+    },
+    "type": "DOCKER"
+  },
+  "requirePorts": true,
+  "mem": 128,
+  "cpus": 0.1,
+  "instances": 1,
+  "id": "/helloworld-universe"
+}
```
Comment (jsancio) [inline cli/tests/data/helloworld-v3-stub.json]: This is one of the important changes. This adds a Universe repo that contains a helloworld package.

File: cli/tests/data/universe-v3-stub.json [hunk 2]
```diff
@@ -26,14 +26,14 @@
...
     },
     "type": "DOCKER"
   },
   "requirePorts": true,
   "mem": 128,
-  "cpus": 0.25,
+  "cpus": 0.1,
   "instances": 1,
   "id": "/universe"
 }
```
Comment (jsancio) [inline cli/tests/data/universe-v3-stub.json]: This is one of the important changes. Update the Universe repo to the latest set of packages.

File: cli/binary/Dockerfile.linux-binary [hunk 3]
```diff
@@ -11,11 +11,17 @@ RUN apt-get update && apt-get install -y \
...
+  language-pack-en \
 && sudo apt-get update --fix-missing \
 && sudo apt-get install -y build-essential libssl-dev libffi-dev python-dev python3-pip python3-venv \
 && pip3 install pip --upgrade \
 && python3 -m pip install pyinstaller==3.1.1
 
+ENV LC_CTYPE en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US
+ENV LC_ALL ""
```
Comment (jsancio) [inline cli/binary/Dockerfile.linux-binary]: This is how you enable "utf-8" encoding for Ubuntu.

File: cli/tests/integrations/test_task.py [hunk 4]
```diff
@@ -184,6 +184,9 @@ def test_log_two_tasks():
 
     lines = stdout.decode('utf-8').split('\n')
     assert len(lines) == 19
+    assert len(
+        [line for line in lines if line.startswith('===> task:test-app')]
+    ) == 2
```
Comment (cruhland) [inline cli/tests/integrations/test_task.py]: What is this for?

File: cli/tests/integrations/test_package.py [hunk 5]
```diff
@@ -265,20 +308,6 @@ def test_describe_app_cli():
...
-    returncode_, stdout_, stderr_ = exec_command(
-        ['dcos', 'package', 'describe', '--package-version=0.11.1',
-         'marathon'])
-
-    assert returncode_ == 0
-    output = json.loads(stdout_.decode('utf-8'))
-    assert output == json.loads(stdout.decode('utf-8'))
-    assert stderr_ == b''
-
-
```
Comment (cruhland) [inline cli/tests/integrations/test_package.py]: Why did this test get deleted?

File: cli/tests/integrations/test_task.py (hunk 4 above)
Comment (jsancio) [inline cli/tests/integrations/test_task.py]: This is a small improvement to the existing test. When logging more than one task the CLI prepends "===> task:{task-name} ..." before the logs of each task.

File: cli/tests/integrations/test_package.py (hunk 5 above)
Comment (jsancio) [inline cli/tests/integrations/test_package.py]: Good catch. This was a duplicate test. `test_describe_package_version` tests the same thing.

**Comment by jsancio** [comment]
Thanks for the review @cruhland !

**Comment by jsancio** [comment]
@tamarrow Can you take a look at this when you get a chance?

**Comment by tamarrow** [comment]
@jsancio can you please update the description + explain the reason we needed to add the utf-8 encoding changes? We should update the README with the necessary CI changes.

**Comment by jsancio** [comment]
> We should update the README with the necessary CI changes.

@tamarrow Where do you want me to add that? Initialization? Also, note that doing that is platform specific. I only know how to do it for Linux. How did you configure it for MacOS and Windows?

**Pull request merged**